RXSwiftTest/Networking/API.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  API.swift
+//  RXSwiftTest
+//
+//  Created by Sunith B on 30/08/19.
+//  Copyright © 2019 Sunith B. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+struct Endpoint {
+    
+    private struct Domains {
+        static let Dev = "https://interviewer-api.herokuapp.com"
+        static let Local = ""
+        static let QA = ""
+    }
+    
+    private struct Routes {
+        static let Api = ""
+    }
+    
+    private static let Domain = Domains.Dev
+    private static let Route = Routes.Api
+    private static let BaseURL = Domain + Route
+    
+    static func login() -> URLRequest? {
```

Comment:
login should take the user credentials as a parameter so that you can construct the request. Instead of creating an empty one